### DISCUSSION TITLE
A search box that returns results from two distinct MkDocs sites

### DISCUSSION THREAD

**Discussion by chr-hen** [squidfunk/mkdocs-material#4046] [Q&A]
Hi there.

I have multiple distinct MkDocs sites, and I link to each of them from a central index page. Currently, this index page is a simple HTML page thay contains only text and hyperlinks.

I'm wondering whether it would be possible to somehow place a search box on that index page which returns results from across my distinct MkDocs sites?

I'm not sure what would be the best approach to achieve this, and whether it would be at all possible.

Thank you.

**Comment by squidfunk** [comment]
There's no plugin I'm aware of, but you could post-process the `search_index.json` of both projects which is generated by MkDocs, in order to merge them. You probably need to resolve relative URLs to be absolute though.
